vfs.go

```diff
@@ -82,6 +82,21 @@ func (v *VFS) Free() {
 	}
 }
 
+// Config retrieves a copy of the config from vfs
+func (c *VFS) Config() (*Config, error) {
```

Comment:
Can you change this to `(v *VFS)`, golang suggests you use the first letter of the struct type lower-cased for struct function names.

Reply:
fixed